web-registry/src/lib/metadata-graph.ts

```diff
@@ -1,11 +1,19 @@
 import axios from 'axios';
+import * as jsonld from 'jsonld';
 
 import getApiUri from './apiUri';
 
 export const getMetadata = async (iri: string): Promise<any> => {
   if (!iri) throw new Error('No metadata iri provided');
   const { data } = await axios.get(`${getApiUri()}/metadata-graph/${iri}`);
-  return data;
+  const compacted = await jsonld.compact(data, {
+    schema: 'http://schema.org/',
+    regen: 'http://regen.network/',
+    qudt: 'http://qudt.org/schema/qudt/',
+    unit: 'http://qudt.org/vocab/unit/',
+    xsd: 'http://www.w3.org/2001/XMLSchema#',
```

Comment:
let's add the snippet of geojson context here as well (we might want to consider deduping this context later)

Reply:
made the context into a resuable const, `COMPACTED_CONTEXT`